packages/connect-examples/expo-playground/app/routes/chains.$chainId._index.tsx

```
@@ -99,87 +110,76 @@ const ChainMethodsIndexPage: React.FC = () => {
+                {filteredMethods.map((method, methodIndex) => (
+                  <div
+                    key={`${selectedChain.id}-${method.method}-${methodIndex}`}
+                    className="group relative overflow-hidden bg-card border border-border/50 rounded-xl p-4 cursor-pointer transition-all duration-300 hover:border-border hover:shadow-md hover:shadow-black/5 dark:hover:shadow-white/5"
+                    onClick={() => handleMethodSelect(method.method)}
+                    onKeyDown={e => handleKeyDown(e, () => handleMethodSelect(method.method))}
+                    tabIndex={0}
+                    role="button"
+                    aria-label={`Execute ${method.method}`}
+                  >
+                    {/* 微妙的悬停效果 */}
+                    <div className="absolute inset-0 bg-gradient-to-r from-foreground/0 via-foreground/0 to-foreground/[0.02] opacity-0 group-hover:opacity-100 transition-opacity duration-300" />
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)